### PR TITLE
chore(*): Java codestyle enforcement

### PR DESCRIPTION
**Pull request by robzienert**
Latest codestyles based on https://github.com/spinnaker/spinnaker-gradle-project/pull/64

### CODE REVIEW

File: kork-jedis/src/main/java/com/netflix/spinnaker/kork/jedis/RedisClientConfiguration.java [hunk 1]
```diff
@@ -132,21 +135,23 @@ private void createDefaultClientIfNotExists(List<RedisClientDelegate> clients,
...
   }
 
   private RedisClientDelegateFactory<?> getClientFactoryForDriver(Driver driver) {
     return clientDelegateFactories.stream()
-      .filter(it -> it.supports(driver))
+      .filter(
+        it -> it.supports(
+          driver
+      )
+        )
```
Comment (robzienert) [inline kork-jedis/src/main/java/com/netflix/spinnaker/kork/jedis/RedisClientConfiguration.java]: This is odd... will try to track down what's doing this.

File: kork-core/src/main/java/com/netflix/spinnaker/kork/lock/LockManager.java [hunk 2]
```diff
@@ -188,22 +185,24 @@ public String getAttributes() {
...
+        ", attributes='" + attributes + '\'' +
+        '}';
     }
 
     @Override
     public boolean equals(Object o) {
       // Two locks are identical if the lock name, owner and version match.
-      if (this == o) return true;
-      if (o == null || getClass() != o.getClass()) return false;
+      if (this == o)
```
Comment (emjburns) [inline kork-core/src/main/java/com/netflix/spinnaker/kork/lock/LockManager.java]: oh god this needs `{}`

Comment (emjburns) [reply kork-core/src/main/java/com/netflix/spinnaker/kork/lock/LockManager.java]: is that a setting? no ifs without {}?

File: kork-expressions/src/main/java/com/netflix/spinnaker/kork/expressions/ExpressionTransform.java [hunk 3]
```diff
@@ -245,8 +247,8 @@ private boolean isExpression(Object obj) {
     return map
       .entrySet()
       .stream()
-      .filter(it ->
-        flatten(it.getValue()).collect(toSet()).stream().flatMap(Stream::of).collect(toSet()).contains(value)
+      .filter(
+        it -> flatten(it.getValue()).collect(toSet()).stream().flatMap(Stream::of).collect(toSet()).contains(value)
```
Comment (emjburns) [inline kork-expressions/src/main/java/com/netflix/spinnaker/kork/expressions/ExpressionTransform.java]: can we change this to keep the named parameter on the first line?

File: kork-expressions/src/main/java/com/netflix/spinnaker/kork/expressions/whitelisting/MapPropertyAccessor.java [hunk 4]
```diff
@@ -30,10 +30,9 @@ public MapPropertyAccessor(boolean allowUnknownKeys) {
 
   @Override
   public boolean canRead(
-    final EvaluationContext context,
-    final Object target,
-    final String name
-  ) throws AccessException {
+                         final EvaluationContext context,
```
Comment (emjburns) [inline kork-expressions/src/main/java/com/netflix/spinnaker/kork/expressions/whitelisting/MapPropertyAccessor.java]: seems inconsistent with functions - is the first param on the same line as the declaration? or on the second line like it is here? Why does it indent all the way?

**Comment by emjburns** [comment]
I ❤️  this

File: kork-core/src/main/java/com/netflix/spinnaker/kork/lock/LockManager.java (hunk 2 above)
Comment (robzienert) [reply kork-core/src/main/java/com/netflix/spinnaker/kork/lock/LockManager.java]: Yes, I'll make it so.

File: kork-expressions/src/main/java/com/netflix/spinnaker/kork/expressions/ExpressionTransform.java (hunk 3 above)
Comment (robzienert) [reply kork-expressions/src/main/java/com/netflix/spinnaker/kork/expressions/ExpressionTransform.java]: Not sure, the eclipse formatter isn't as good as the intellij one. I'd 100% love to get a command-line intellij formatter in.

File: kork-expressions/src/main/java/com/netflix/spinnaker/kork/expressions/whitelisting/MapPropertyAccessor.java (hunk 4 above)
Comment (robzienert) [reply kork-expressions/src/main/java/com/netflix/spinnaker/kork/expressions/whitelisting/MapPropertyAccessor.java]: 🤔 yeah this is very wrong. Nice spot!

**Comment by robzienert** [comment]
Travis is being dumb. Opening a new PR

**Pull request closed without merging**